From 645271f1386800d982332e15b3ebc817ad4b6f94 Mon Sep 17 00:00:00 2001
From: Ahmed Al-Nassif <mr.ahmed.nassif@gmail.com>
Date: Mon, 13 Jul 2026 10:41:24 +0300
Subject: [PATCH 02/24] fix: Remove ifdef wrapping of key_pass declaration

key_pass is used unconditionally later in the file but was incorrectly
wrapped in #ifdef USE_PKCS11_ENGINE during merge resolution, causing
build failures when PKCS11 engine support is not enabled.

Signed-off-by: Ahmed Al-Nassif <mr.ahmed.nassif@gmail.com>
---
 certs/extract-cert.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/certs/extract-cert.c b/certs/extract-cert.c
index 35fa2470d..d68975f68 100644
--- a/certs/extract-cert.c
+++ b/certs/extract-cert.c
@@ -77,9 +77,7 @@ static void drain_openssl_errors(void)
 		}					\
 	} while(0)
 
-#ifdef USE_PKCS11_ENGINE
 static const char *key_pass;
-#endif
 static BIO *wb;
 static char *cert_dst;
 static int kbuild_verbose;
-- 
2.55.0

